src/nupic/algorithms/TemporalMemory.hpp

```diff
@@ -631,6 +630,7 @@ using namespace nupic::algorithms::connections;
 
 public:
   Connections connections; //TODO not public!
+  UInt extra_;
```

Comment:
can you find other way around for tha bindings than making the extra_ public? If it's public, it shouldn't be world-writable, make const&?

Reply:
I made this a const &